data/changelog.txt

```diff
@@ -397,6 +413,11 @@ Allow ebooks in read command
 format `uilist` menu entries with a table
 Prevent automatically assigning keys to passive mutations
 Fix ImGui Demo in curses
+Fix ImGui Demo in curses
+Fixes errors and discrepancies in spelling, grammar, punctuation, and capitalization
+make ImTui render text as text, so that the players can read our spiffy new menus
+toggle to only draw overmap during autotravel
+mouse input in uilists does not block keyboard navigation
```

Comment:
```suggestion
make ImTui render text as text, so that the players can read our spiffy new menus
toggle to only draw overmap during autotravel
mouse input in uilists does not block keyboard navigation
```